cryptol-remote-api/python/cryptol/opaque.py

```diff
@@ -1,5 +1,7 @@
 from typing import Any
+from dataclasses import dataclass
 
+@dataclass(frozen=True)
```

Comment:
Can we get a comment why this is `frozen` but other `dataclass`es are not?

Reply:
Are you satisfied by the comment I added?